patch/netlink-socket-attribute-filter.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/include/uapi/linux/bpf.h b/include/uapi/linux/bpf.h
```

Comment:
Please use a `git format-patch` formatted patch, which can be easily applied with `git am ….patch` to the Linux kernel source. Please add a descriptive commit summary, message and Signed-off-by line.

Reply:
Fixed